lib/dailymotion-api/client.rb

```diff
@@ -46,31 +55,35 @@ def get_video(video_id, fields = "")
       return nil unless video_id
 
       url = "#{API_URL}/video/#{video_id}"
-      url << "?fields=#{fields}" unless fields.empty?
+      url += "?fields=#{fields}" unless fields.empty?
```

Comment:
Because `frozen_string_literal` does not allow you to do so any more. And yes, you could avoid creating 2 String objects, but it is not better, if you do so only ones.


https://freelancing-gods.com/2017/07/27/friendly-frozen-string-literals.html